### PR TITLE
fix: remove generated folders and fix storybook

### PR DESCRIPTION
**Pull request by wesbillman**

### CODE REVIEW

**Review by deniseli — APPROVED** [review]
awesome thanks :D

File: frontend/console/.storybook/main.ts
Comment (alecthomas) [inline frontend/console/.storybook/main.ts:1]: I don't think these directories/files are in .gitignore?

Comment (wesbillman) [reply frontend/console/.storybook/main.ts:1]: `.storybook` should not be ignored. Just the `playwright-report` and `test-results`

Comment (alecthomas) [reply frontend/console/.storybook/main.ts:1]: Ah I see! I just missed it because it was stealth mode!

**Pull request merged**